bin/openshift-include.sh

```diff
@@ -64,12 +64,12 @@ function ocVersionCheck {
     ### Openshift versions that do not start with a 4 should produce an error.
     if (( "$OC_MAJOR_VERSION" != 4 )); then
       log_error "Unsupported OpenShift client version: $OC_FULL_VERSION"
-      log_error "Version 4.8+ is required"
+      log_error "Version 4.9+ is required"
```

Comment:
No real issue just think its interesting that the error msg is the same as the one below, so maybe at somepoint we should just combine the version checks into a single IF block.